src/DependencyInjection/Factory/BlockAdminFactory.php

```diff
@@ -70,10 +73,12 @@ public function create(array $config, ContainerBuilder $container, XmlFileLoader
 
         $bundles = $container->getParameter('kernel.bundles');
 
-        if (true === $config['enable_menu']
-            || ('auto' === $config['enable_menu'] && isset($bundles['CmfMenuBundle']))
+        if ($this->isConfigEnabled($container, $config, 'enable_menu')
+            && array_key_exists('CmfMenuBundle', $bundles)
         ) {
             $loader->load('block-menu.xml');
+        } elseif (true === $config['enable_menu'] && !array_key_exists('CmfMenuBundle', $bundles)) {
```

Comment:
do we need to resolve the value here as well? but we need to distinguish between `'auto'` and `true`